appveyor.yml

```diff
@@ -67,20 +63,23 @@ deploy:
       branch: dev
 
   - provider: GitHub
+    repository: Flow-Launcher/Prereleases
     release: v$(flowVersion)
+    description: 'This is a release candidate of v$(flowVersion)'
     auth_token:
       secure: ij4UeXUYQBDJxn2YRAAhUOjklOGVKDB87Hn5J8tKIzj13yatoI7sLM666QDQFEgv
     artifact: Squirrel Installer, Portable Version, Squirrel nupkg, Squirrel RELEASES
-    draft: true
     force_update: true
     on:
       branch: master
+      APPVEYOR_REPO_TAG: false
 
   - provider: GitHub
     release: v$(flowVersion)
     auth_token:
       secure: ij4UeXUYQBDJxn2YRAAhUOjklOGVKDB87Hn5J8tKIzj13yatoI7sLM666QDQFEgv
     artifact: Squirrel Installer, Portable Version, Squirrel nupkg, Squirrel RELEASES
-    force_update: true
```

Comment:
We have never had to do a hot fix during release process, but in the case we that we do have to, this would mean it will not override the release artefacts right? I suppose you need to delete the draft then commit the hot fix to master in order for the new artefacts to get uploaded.